### ISSUE TITLE
Kotlin compiler plugin isn't applied when using embeddable compiler jars

### ISSUE THREAD

**Issue by filimo** [rickclephas/KMP-NativeCoroutines#67]
I had build.gradle.kts
```
id("com.rickclephas.kmp.nativecoroutines") version("0.12.2-new-mm")
....
classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21")
classpath("org.jetbrains.kotlin:kotlin-serialization:1.6.21")
```

When I upgraded build.gradle.kts  to 
```
id("com.rickclephas.kmp.nativecoroutines") version("0.12.3-new-mm")
....
classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.0")
classpath("org.jetbrains.kotlin:kotlin-serialization:1.7.0")
```
My KMM Library build is ok but Xcode project can't find native functions.
```
error: value of type 'SharedRepository' has no member 'getDisplayStringsNative'
```

**Comment by rickclephas** [comment]
Thanks for reporting this @filimo!
I did a local test and got similar errors for the sample project. However the tests (which use the sample project) are passing on GH actions. So it doesn't seem entirely related to the Kotlin/KMP-NativeCoroutines update.

Could you provide some details about your setup? Like your Xcode version and if you are using an Apple Silicon device?

**Comment by rickclephas** [comment]
Could you try and add the following to your `gradle.properties` file?:
```properties
kotlin.native.useEmbeddableCompilerJar=false
```

**Comment by rickclephas** [comment]
Fixed in [v0.12.4](https://github.com/rickclephas/KMP-NativeCoroutines/releases/tag/v0.12.4).

**Comment by filimo** [comment]
Thanks @rickclephas for your very quick fix. Great job!. 

I tested on Xcode 13.4.1 and 14.0 beta is ok. 
I have M1 Max CPU.
